lib/template-mixins.js

```diff
@@ -514,6 +514,14 @@ module.exports = function (options, deprecated) {
             };
         };
 
+        res.locals.renderField = function () {
+            return function () {
+                var mixin = this.mixin;
+                if (mixin && res.locals[mixin] && typeof res.locals[mixin] === 'function') {
+                    return res.locals[mixin]().call(Object.assign({}, res.locals, this), this.key);
+                }
```

Comment:
Do we want to do anything if the mixin specified is not found? At least log a warning maybe?